src/points/PointsHome.tsx

```diff
@@ -0,0 +1,134 @@
+import { RouteProp } from '@react-navigation/native'
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
+import { StackParamList } from 'src/navigator/types'
+import BackButton from 'src/components/BackButton'
+import React, { useEffect, useState, useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+import { emptyHeader } from 'src/navigator/Headers'
+import { Screens } from 'src/navigator/Screens'
+import { PointsEvents } from 'src/analytics/Events'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { Colors } from 'src/styles/colors'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import ActivityCardSection from 'src/points/ActivityCardSection'
+import BottomSheet, { BottomSheetRefType } from 'src/components/BottomSheet'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import { BottomSheetDetails } from 'src/points/types'
+
+type Props = NativeStackScreenProps<StackParamList, Screens.PointsHome>
+
+export default function PointsHome({ route, navigation }: Props) {
+  const { t } = useTranslation()
+
+  // TODO: Use real points balance
+  const pointsBalance = 50
+
+  const bottomSheetRef = useRef<BottomSheetRefType>(null)
```

Comment:
Getting the bottom sheet to work was a little tricky, so the implementation here might seem a bit wonky... basically, we'll want to support _arbitrarily many_ bottom sheets on this page, one for each pressable card. Rather than dynamically generating refs equal to the number of pressable cards and adding a `BottomSheet` component for each one, I just decided to use a _single_ ref, and when a card is pressed, pipe the data required to render the bottom sheet back up from where it's defined (from within the cards defined in `src/points/activityCards`) to here. The `useEffect` is here to ensure that we only show the bottom sheet once the state has definitely been set.